spec/integrated/vnext-spec/rows-spec.js

```diff
@@ -35,45 +35,46 @@ let gridConfig = {
     primaryKey: 'UserName',
   },
   rows: {
-    headRows: [
-      {
-        html: '<div class="head-rows-html">head rows html</div>',
+    headRows: [{
+      html: '<div class="head-rows-html">head rows html</div>',
+      payload: 'Custom Row',
+      attributes: {
+        'data-type': 'html',
+        'data-payload': _.property('payload'),
       },
-      'column-header-rows',
-    ],
+    }, 'column-header-rows'],
     bodyRows: [
       {
         type: 'data-rows',
         classes: {
-          male: (row) => {
-            return row.Gender === 'Male';
-          },
-          female: (row) => {
-            return row.Gender === 'Female';
-          },
+          male: row => (row.Gender === 'Male'),
+          female: row => (row.Gender === 'Female'),
+        },
+        attributes: {
+          'data-type': 'data',
+          'data-first-name': _.property('FirstName'),
         },
       },
     ],
-    footRows: [
-      {
-        view: new TitleView({title: 'view in footer'}).render(),
+    footRows: [{
+      view: new TitleView({ title: 'view in footer' }).render(),
+      attributes: {
+        'data-type': 'view',
+        'data-title': ({ view }) => view.title,
       },
-    ]
+    }],
   },
 };
 
-let pgrid;
-let gridView;
+let pgrid, gridView;
```

Comment:
per the eslint rule we are using, if the variables are not initialized, we should put them in 1 line.